### ISSUE TITLE
Disable meal open/close functionality to non-chefs

### ISSUE THREAD

**Issue by crw** [joyvuu-dave/comeals-ui#4]
Random non-cooks keep opening and closing meals. This should be restricted to the chefs who are cooking. 

There is an issue when a chef doesn't log into their account and needs to use the ipad account. Maybe this can be an options on a chef-by-chef basis.

**Comment by joyvuu-dave** [comment]
I agree it's totally annoying if someone opens a previously closed meal and adds themselves. One of the reasons it didn't seem necessary to prevent non-cooks from opening / closing was that we log the history of all actions. It's not very prominent, but there is a history button in the top right corner of the meals page .So there should never be an issue about _if_ someone added themselves after a meal was closed. Just that fact that they can.

That being said, if the UI made sense, I wouldn't foresee it being too much of an issue to add this feature. How do you picture the flow working for meals with multiple cooks?

**Comment by crw** [comment]
The log doesn't help as some members are beyond shame. :D 

For multiple cooks, if each and every cook had the "lock open/close" flag set, only those cooks would be able to open/close. If any one of the cooks did not, anyone could open/close.

**Comment by stappon** [comment]
If we add such a flag, we could also use it to prevent other people from editing your meal sign-up by mistake. I've accidentally signed up other people for meals in the past by fat-fingering the wrong row when trying to add guests etc to my own sign-up.
